src/contrib/cluster/Akka.Cluster.Metrics/reference.conf

```diff
@@ -0,0 +1,111 @@
+        # Child actor restart-on-failure window.
+        withinTimeRange = 3s
+        # Maximum number of restart attempts before child actor is stopped.
+        maxNrOfRetries = 3
+      }
+    }
+  }
+  # Metrics collector actor.
+  collector {
+    # Enable or disable metrics collector for load-balancing nodes.
+    # Metrics collection can also be controlled at runtime by sending control messages
+    # to /system/cluster-metrics actor: `akka.cluster.metrics.{CollectionStartMessage,CollectionStopMessage}`
+    enabled = on
+    # FQCN of the metrics collector implementation.
+    # It must implement `akka.cluster.metrics.MetricsCollector` and
+    # have public constructor with akka.actor.ActorSystem parameter.
+    # Will try to load in the following order of priority:
+    # 1) configured custom collector 2) internal `SigarMetricsCollector` 3) internal `JmxMetricsCollector`
+    provider = ""
+    # Try all 3 available collector providers, or else fail on the configured custom collector provider.
+    fallback = true
+    # How often metrics are sampled on a node.
+    # Shorter interval will collect the metrics more often.
+    # Also controls frequency of the metrics publication to the node system event bus.
+    sample-interval = 3s
+    # How often a node publishes metrics information to the other nodes in the cluster.
+    # Shorter interval will publish the metrics gossip more often.
+    gossip-interval = 3s
+    # How quickly the exponential weighting of past data is decayed compared to
+    # new data. Set lower to increase the bias toward newer values.
+    # The relevance of each data sample is halved for every passing half-life
+    # duration, i.e. after 4 times the half-life, a data sample’s relevance is
+    # reduced to 6% of its original relevance. The initial relevance of a data
+    # sample is given by 1 – 0.5 ^ (collect-interval / half-life).
+    # See http://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average
+    moving-average-half-life = 12s
+  }
+}
+
+# Cluster metrics extension serializers and routers.
+akka.actor {
+  # Protobuf serializer for remote cluster metrics messages.
+  serializers {
+    akka-cluster-metrics = "Akka.Cluster.Metrics.Serialization.ClusterMetricsMessageSerializer"
+  }
+  # Interface binding for remote cluster metrics messages.
+  serialization-bindings {
+    "akka.cluster.metrics.ClusterMetricsMessage" = akka-cluster-metrics
+    "akka.cluster.metrics.AdaptiveLoadBalancingPool" = akka-cluster-metrics
+    "akka.cluster.metrics.MixMetricsSelector" = akka-cluster-metrics
+    "akka.cluster.metrics.CpuMetricsSelector$" = akka-cluster-metrics
+    "akka.cluster.metrics.HeapMetricsSelector$" = akka-cluster-metrics
+    "akka.cluster.metrics.SystemLoadAverageMetricsSelector$" = akka-cluster-metrics
+  }
+  # Globally unique metrics extension serializer identifier.
+  serialization-identifiers {
+    "akka.cluster.metrics.protobuf.MessageSerializer" = 10
```

Comment:
This namespace is wrong - needs to be a CLR fully-qualified type name. Also, is the number 10 taken already? I don't think so since this functionality was originally built into Akka.Cluster, but it'd be good to check (although I guess that will come up when you start porting the multi-node tests.)

Reply:
You know, that's funny - 7/8/9 are taken, 11/12/13 are taken, but 10 is not (searching in configurations by `serialization-identifiers` string). So will keep it like that, and will see what multi-node tests will show.
Thanks for type name hint - fixed it

Reply:
Sounds like this is resolved then.